package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "@resin/lf-to-abstract-sql",
+  "name": "@balena/lf-to-abstract-sql",
   "version": "3.2.6",
   "description": "LF to Abstract SQL translator.",
   "main": "index.js",
   "scripts": {
-    "lint": "balena-lint test",
+    "lint": "balena-lint test && balena-lint --typescript index.js",
+    "lint-fix": "balena-lint --typescript --fix index.js",
     "pretest": "npm run prepare",
     "test": "mocha",
+    "posttest": "npm run lint",
     "prepublish": "require-npm4-to-publish",
```

Comment:
How about dropping this as well as legacy ? :)

Reply:
My thoughts on it pretty much boil down to it avoids any possible issues and doesn't really add overhead to keep, I expect I'll change my mind a bit further down the line as we get even further away from the world where older npm might exist but for now I don't see a need